Xamarin.PropertyEditing.Mac/HostResourceProvider.cs

```diff
@@ -19,6 +19,11 @@ public HostResourceProvider ()
 		{
 			var containingDir = Path.GetDirectoryName (typeof (HostResourceProvider).Assembly.Location);
 			var bundlePath = Path.Combine (containingDir, "PropertyEditingResource.bundle");
+			if (!Directory.Exists(bundlePath))
+			{
+				//if the directory is not in place we fallback into bundle resources directory
+				bundlePath = NSBundle.MainBundle.PathForResource ("PropertyEditingResource", "bundle");
```

Comment:
Which one is more likely to happen? Maybe we can invert the two.

Reply:
fixed